robot_skills/src/robot_skills/arms.py

```diff
@@ -455,6 +469,40 @@ def wait_for_motion_done(self, timeout=10.0, cancel=False):
             # return self._ac_gripper.wait_for_result(rospy.Duration(timeout - passed_time))
             return True
 
+    @property
+    def object_in_gripper_state(self):
```

Comment:
The states that are using these are responsible for this. This addition is a reflection of the measured gripper state. Should be update the name?

Reply:
Renamed to object in gripper measurement in 83184c2e8363a998b91e3b7a6d04d9bec7e4a3d5 . This should be more clear since this reflects a measurement and the state you're talking about is the internal state of this Python session.